README.md

```diff
@@ -1,31 +1,332 @@
+
+NFL refs receive Super Bowl rings. (2 points)
+(1) True
+(2) False
+
+Olympic gold metals are made of silver. (3 points)
+(1) True
+(2) False
+```
+
+Create a subclass of `Question` named `MultipleChoiceQuestion`.
+
+- Add an instance variable named `choices` that stores an array of strings representing the multiple choices. Add set/get methods for `choices`.
+- Override the `displayQuestion()` method with the following logic:
+    - Call the superclass `displayQuestion()` method to print the question text and points on one line
+    - Use a loop to print each array element as a numbered choice on a separate line as shown:
+
+```text
+Which team scored the most points in the 2018 season? (5 points)
+(1) Patriots
+(2) Broncos
+(3) Browns
+(4) Steelers
+```
+
+Edit `Task2Main` to create two `MultipleChoiceQuestion` objects and display as shown:
+
+```java
+public class Task2Main {
+	public static void main(String []args)  {
+		Question q1 = new Question();
+		q1.setText("Which sport has been played on the moon?");
+		q1.setPoints(7);
+		q1.displayQuestion();
+
+		System.out.println();
+		TrueFalseQuestion q2 = new TrueFalseQuestion();
+		q2.setText("NFL refs receive Super Bowl rings.");
+		q2.setPoints(2);
+		q2.displayQuestion();
+
+		System.out.println();
+		TrueFalseQuestion q3 = new TrueFalseQuestion();
+		q3.setText("Olympic gold metals are made of silver.");
+		q3.setPoints(3);
+		q3.displayQuestion();
+
+		System.out.println();
+		MultipleChoiceQuestion q4 = new MultipleChoiceQuestion();
+		q4.setText("Which team scored the most points in the 2018 season?");
+		q4.setPoints(5);
+		q4.setChoices(new String[]{"Patriots", "Broncos", "Browns", "Steelers"});
+		q4.displayQuestion();
+
+		System.out.println();
+		MultipleChoiceQuestion q5 = new MultipleChoiceQuestion();
+		q5.setText("What sport did James Naismith invent?");
+		q5.setPoints(10);
+		q5.setChoices(new String[]{"Football", "Basketball", "Baseball"});
+		q5.displayQuestion();
+	}
+}
+```
+
+Confirm the output:
+
+```text
+Which sport has been played on the moon? (7 points)
+
+NFL refs receive Super Bowl rings. (2 points)
+(1) True
+(2) False
+
+Olympic gold metals are made of silver. (3 points)
+(1) True
+(2) False
+
+Which team scored the most points in the 2018 season? (5 points)
+(1) Patriots
+(2) Broncos
+(3) Browns
+(4) Steelers
+
+What sport did James Naismith invent? (10 points)
+(1) Football
+(2) Basketball
+(3) Baseball
+```
+
+
+## Task #3
+
+You will implement the following interface hierarchy:
+
+![task3 uml](https://curriculum-content.s3.amazonaws.com/6677/pillars/task3_uml.png)
+
+Create a new interface named `Flippable` with one abstract method named `flip()` that
+takes no parameters and has a `void` return type.
+
+Create a new class named `Coin` that implements `Flippable`.  The `flip()` method should
+print a random string of "Heads" or "Tails" each time it is called.
+
+Create a new class named `Mattrest` that implements `Flippable`. Add a boolean instance
```

Comment:
Typo: `Mattrest` should be `Mattress`

But I like how you tried to put the word "rest" in it haha

Reply:
:-)